web/src/components/gnd-main/index.js

```diff
@@ -16,10 +16,24 @@
  */
 
 import React from "react";
+import { Route } from 'react-router' // react-router v4
 import GndMap from "../gnd-map";
 import GndHeader from "../gnd-header";
 import GndLegend from "../gnd-legend";
 import GndFeatureTypeEditor from "../gnd-feature-type-editor";
+import { connectProject, connectFeature } from "../../datastore.js";
+import { featurePath } from '../../route.js'
+
+
+function GndObservations({ match }) {
```

Comment:
Naming nit: Maybe this should be `GndFeatureDetails`? More realistically this will live inside another component that represents the side panel, but out of scope of this PR.

Reply:
Yes, I used it as a placeholder and just to show that it is possible to get projectId and featureId from the match